modules/openapi-generator-online/Dockerfile

```diff
@@ -4,9 +4,10 @@ WORKDIR /generator
 
 COPY target/openapi-generator-online.jar /generator/openapi-generator-online.jar
 
-ENV GENERATOR_HOST=http://localhost
+# GENERATOR_HOST can be used to determine the target location of a download link.
+# The default value asumes binding to host via: docker -p 8080:8080 image_name
+ENV GENERATOR_HOST=http://localhost:8080
```

Comment:
I'm not sure I understand. Do we keep GENERATOR_HOST?

Reply:
@cbornet `GENERATOR_HOST` is still used here:

https://github.com/OpenAPITools/openapi-generator/blob/109808e60d15e97680f274bda8ed854da52b67ea/modules/openapi-generator-online/src/main/java/org/openapitools/codegen/online/service/GenApiService.java#L161

This isn't "correct" and will need to be changed. The previous value here would be incorrect for all invocations unless bound to port 80, which I believe requires super user privileges. This change at least causes the variable to match the exposed port.

Many people try out docker images by binding to random host ports (`-P`), and when doing this… `GENERATOR_HOST` will be useless and the file download link generation won't work.

This PR limits focus to get the proxying host setting working on the server. I can look at the other work later.

Reply:
Created #3288 to address this other concern.